packages/cdc-react/src/components/Sidebar/SidebarSection/SidebarSection.tsx

```diff
@@ -22,9 +22,11 @@ export const SidebarSection = ({
       {heading && !hideLabels && <p className="heading">{heading}</p>}
       <ul>
         {items.map((item, index) => (
-          <li key={`${item.text}${index}`} role="button">
-            <Icon className={`${item.icon}-icon icon`} name={item.icon} />
-            {!hideLabels && <span className="text">{item.text}</span>}
+          <li key={`${item.text}${index}`}>
+            <a href="">
```

Comment:
Discussion for another day: How will we be able to integrate these anchor tags with out front end routing.